quiz.js

```diff
@@ -0,0 +1,75 @@
+// Create a tree function that should build a pine tree out of a character in the Chrome dev tools console.
+
+// It accepts a single object as an argument. The object should have two key/value pairs.
+
+// A key that specifies the height of the pine tree.
+
+// The value for the height of the tree should be from user input in a <input type="text"> field in the DOM.
+
+// A key that specifies which character to use to build the pine tree.
+
+// The character to use should be from user input in a <input type="text"> field in the DOM.
+
+
+var tree = {
+	height: userHeight,
+	character: userChar
+};
+
+
+// function growTree(height, char) {
+// 	// body... call variables inside so they don't come up as undefined
+// }
+
+
+
+var treeButton = document.getElementById("yourTree");
+
+
+
+
+function checkInput(yes) {
+	var height = document.getElementById("userHeight").value;
+	var char = document.getElementById("userChar").value;
+	if (isNaN(height)  || (char.length > 1 || char.length < 1)) {
+	alert("Please make sure you enter a number for height and a character before trying to grow your tree");
+// If either of the input fields does not have a value in it when the user presses the enter key, or presses the button, then display an alert stating that both fields must have a value.
+	}
+	else {
+		growTree(height, char);
+	}
+}
+
+
+function growTree(height, char) {
```

Comment:
Here you are passing the properties of the object. Instead you should be passing the entire object.